index.html

```diff
@@ -12,74 +12,130 @@
 <body>
   <div class="container">
     <header>
-      <p class="logo">Henry Ik</p>
-      <nav>
-        <img src="images/Union.png" alt="menu" />
-      </nav>
+      <div class=" nav mobile">
+        <p class="logo">Henry Ik</p>
+        <nav>
+          <img src="images/Union.png" alt="menu" />
+        </nav>
+      </div>
+
+      <div class=" nav desktop">
+        <p>Henry Ik</p>
+        <nav>
+          <ul>
+            <a href="#">
+              <li>Portfolio</li>
+            </a>
+            <a href="">
+              <li>About</li>
+            </a>
+            <a href="">
+              <li>Contact</li>
+            </a>
+            <a href=""><img src="images/envelope.png" alt="envelope"></a>
+          </ul>
+        </nav>
+      </div>
```

Comment:
Good job! but it's highly recommended to follow the exact Figma design, so please make sure to fix the font size and weight of your navbar links. Plus the spacing.

[Optional] You're using double `nav` tags but you can put them just in one nav menu and you hide and show them bt the screen size.

Original:
![image](https://user-images.githubusercontent.com/34779928/206297394-d91ededd-2121-493d-8917-bf6106c630e6.png)

Yours
![image](https://user-images.githubusercontent.com/34779928/206297473-a10b3fe8-af14-4a1b-aa49-30ef3664535e.png)